praw/models/reddit/live.py

```diff
@@ -122,6 +122,71 @@ def __init__(self, thread):
         """
         self.thread = thread
 
+    def invite(self, redditor, permissions=None):
+        """Invite a redditor to be a contributor of the live thread.
+
+        Raise :class:`praw.exceptions.APIException` if the invitation
+        already exists.
+
+        :param redditor: A redditor name (e.g., ``'spez'``) or
+            :class:`~.Redditor` instance.
+        :param permissions: When provided (not ``None``), permissions should
+            be a list of strings specifying which subset of permissions to
+            grant. An empty list ``[]`` indicates no permissions, and when
+            not provided (``None``), indicates full permissions.
+
+        Usage:
+
+        .. code-block:: python
+
+           thread = reddit.live('ukaeu1ik4sw5')
+           redditor = reddit.redditor('spez')
+
+           # 'manage' and 'settings' permissions
+           thread.contributor.invite(redditor, ['manage', 'settings'])
+
+        :seealso: :meth:`.LiveContributorRelationship.remove_invite` to
+            remove the invite for redditor.
+
+        """
+        if permissions is None:
```

Comment:
It's not possible because [`'-access'` permission](https://github.com/praw-dev/praw/blob/d418f9b2d872cdc5837e31e0f1c56391bf7b18e5/praw/models/reddit/subreddit.py#L1221) is invalid for live thread contributors. reddit returns an error if specified permission is invalid:

```
$ cat permtest.py
import praw

reddit = praw.Reddit("prawtest")
data = dict(name='nmtake', permissions='-access')
reddit.post('api/live/xyu8kmjvfrww/invite_contributor', data=data)

$ python permtest.py
Traceback (most recent call last):
  File "permtest.py", line 6, in <module>
    reddit.post('api/live/xyu8kmjvfrww/invite_contributor', data=data)
  File "/Users/me/github/praw/praw/reddit.py", line 366, in post
    return self._objector.objectify(data)
  File "/Users/me/github/praw/praw/objector.py", line 51, in objectify
    raise APIException(*errors[0])
praw.exceptions.APIException: INVALID_PERMISSION_TYPE: 'permissions don't apply to that type of user' on field 'type'
```

Valid permissions for reddit live thread contributors are defined [here](https://github.com/reddit/reddit-plugin-liveupdate/blob/a7cde1cf917c4118c4cb114c73a92f272e439816/reddit_liveupdate/permissions.py).